internal/setup/setup.go

```diff
@@ -534,14 +558,6 @@ func parseRelease(baseDir, filePath string, data []byte) (*Release, error) {
 		if len(details.Components) == 0 {
 			return nil, fmt.Errorf("%s: archive %q missing components field", fileName, archiveName)
 		}
-		if len(yamlVar.Archives) == 1 {
-			details.Default = true
-		} else if details.Default && release.DefaultArchive != "" {
-			return nil, fmt.Errorf("%s: more than one default archive: %s, %s", fileName, release.DefaultArchive, archiveName)
```

Comment:
If we do support the suggested compatibility approach, we should fail if we find archives using both "default" and "priority".

Reply:
Yes, if the idea is to use this on older releases, we need to keep both pieces of data. Let's continue on the thread above.